mpf/devices/score_reel_group.py

```diff
@@ -75,7 +75,7 @@ def chime(cls, chime, **kwargs):
         del kwargs
         chime.pulse()
 
-    def set_value(self, value):
+    def set_value(self, value, quiet=False):
```

Comment:
Aligned with my previous comment, changing the argument from `quiet` to `is_reset` will provide usecase-independent context and help future developers understand what's happening.